beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -123,6 +123,7 @@ func (s *Server) GetAggregateAttestationV2(w http.ResponseWriter, r *http.Reques
 		}
 		resp.Data = data
 	}
+	w.Header().Set(api.VersionHeader, version.String(agg.Version()))
```

Comment:
Per https://github.com/ethereum/beacon-APIs/pull/470/files, the version should be `The active consensus version`, not just Phase0/Electra

Reply:
You're right, I'll fix that